graph/telemetry/istio/appender/appender.go

```diff
@@ -180,13 +186,29 @@ func ParseAppenders(o graph.TelemetryOptions) []graph.Appender {
 		appenders = append(appenders, a)
 	}
 
-	return appenders
+	// The finalizer order is important
+	// always run the outsider finalizer
+	finalizers = append(finalizers, &OutsiderAppender{
+		AccessibleNamespaces: o.AccessibleNamespaces,
+		Namespaces:           o.Namespaces,
+	})
+
+	// if labeler finalizer is to be run, do it after the outsider finalizer
+	if _, ok := requestedFinalizers[LabelerAppenderName]; ok {
+		finalizers = append(finalizers, &LabelerAppender{})
+	}
+
+	// always run the traffic generator finalizer
+	finalizers = append(finalizers, &TrafficGeneratorAppender{})
+
+	return appenders, finalizers
 }
 
 const (
 	serviceListKey       = "serviceListKey"       // global vendor info map[namespace]serviceDefinitionList
 	serviceEntryHostsKey = "serviceEntryHostsKey" // global vendor info service entries for all accessible namespaces
 	workloadListKey      = "workloadListKey"      // global vendor info map[namespace]workloadListKey
+	appsMapKey           = "appsMapKey"           // global vendor info map[namespace]appsMap
```

Comment:
alphabetize these constants. not a blocker.

Reply:
yeeeeeees, teacherrrrrr.
done.